.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: ruby
 rvm:
-- "2.1"
-- "2.2.6"
-- "2.3.3"
-- "2.4.0"
+- 2.4
```

Comment:
These should really be strings, otherwise they can cause issues on the Travis side as YAML parsing turns this into floats.

Reply:
They're not strings anywhere else ... https://github.com/ManageIQ/manageiq/blob/master/.travis.yml#L4-L5

Reply:
Those are strings even though they aren't quoted...welcome to YAML parsing.

`key: 2` parses to Integer
`key: 2.4` parses to Float
`key: 2.4.0` parses to String

Reply:
:tired_face: Okay, I can update ... but for the record, travis is clearly fine with these as floats (the spec matrix ran fine).

Reply:
https://github.com/ManageIQ/linux_admin/pull/207

Reply:
> travis is clearly fine with these as floats

That must be a more recent fix, because that always failed in the past...good to know :+1:

Reply:
lol, I didn't know this was a problem... I thought it was a consistency thing

Reply:
Note, the blaster outer needs to do this correctly if we need these to be string literals.  It's currently putting bare values in the .travis.yml.

Reply:
Actually it looks like two digit versions would fail in the blaster outer https://github.com/ManageIQ/manageiq-release/blob/1be60fcafc4bf6ee499f54f853943424e7a7f327/scripts/pull_request_blaster_outer/change_travis_ruby_versions.rb#L4

Reply:
And to @Fryguy's point it looks like it would do the right thing and quote the two digit string:
```
irb(main):010:0> versions
=> ["2.3", "2.4", "2.5"]
irb(main):011:0> puts({"rvm" => versions}.to_yaml)
---
rvm:
- '2.3'
- '2.4'
- '2.5'
=> nil

```

Reply:
yeah, I have to fix up the blaster outer... it's trying to do too much... it should really only be used for version bumps to a bunch of repos... I'll take a look